robot_smach_states/src/robot_smach_states/dummy_state.py

```diff
@@ -0,0 +1,48 @@
+import smach
+import rospy
+
+import robot_smach_states.util.designators as ds
+
+__all__ = ["DummyState"]
+
+
+class DummyState(smach.State):
+    """
+    DummyState to which returns the input outcome
```

Comment:
words are hard to english. My guess is this should say "DummyState which returns the input as outcome"

Reply:
```suggestion
    DummyState which returns the input as the outcome
```